experiments/PlaywrightSharp/PlaywrightSharp/ApiSurface/ITimeoutError.cs

```diff
@@ -48,7 +48,7 @@ namespace PlaywrightSharp
 	/// TimeoutError is emitted whenever certain operations are terminated due to timeout, e.g. <see cref="IPage.WaitForSelectorAsync"/> or
 	/// <see cref="IBrowserType.LaunchAsync"/>.
 	/// </summary>
-	public interface ITimeoutError
+	public interface ITimeoutError : Exception
```

Comment:
This won't compile 🤓

Reply:
I know :-) The project doesn't compile at the moment, I'm working through the problems. I'm moving `ITimeoutError` to be `TimeoutException`. We can reconsider the approach to exceptions once the first "draft" lands, I think.